packages/ui-components/src/components/Atoms/typography/Headline4.tsx

```diff
@@ -0,0 +1,14 @@
+import React, { ReactNode } from "react";
+
+interface Props {
+	children: ReactNode;
+	className?: string;
+}
+
+export const Headline4 = (props: Props) => {
```

Comment:
Just curious, why you never use destructuring?

Reply:
In fact that one is not correct, lol >.<